packages/cli/src/UserManagement/UserManagementHelper.ts

```diff
@@ -78,6 +78,16 @@ export async function getInstanceOwner(): Promise<User> {
 	return owner;
 }
 
+export async function getRole(scope: RoleScopes, name: RoleNames): Promise<Role> {
```

Comment:
You can also use lookup types to avoid ex/importing: `Role['name']`, `Role['scope']`.